chart/compass/charts/system-broker/templates/deployment.yaml

```diff
@@ -58,10 +58,11 @@ spec:
             # - name: SERVER_SHUTDOWN_TIMEOUT
             #   value: "0.0.0.0:{{ .Values.deployment.args.containerPort }}"
             - name: SERVER_SELF_URL
-              value: "https://{{ .Values.global.system_broker.tls.host }}.{{ .Values.global.ingress.domainName }}/broker"
-            - name: GRAPHQL_CLIENT_GRAPHQL_ENDPOINT
+              value: "https://{{ .Values.global.gateway.mtls.host }}.{{ .Values.global.ingress.domainName }}"
+            - name: GRAPHQL_CLIENT_GRAPHQL_ENDPOINT # directly call the compass-gateway, as we already have a crafted jwt id token
+              value: "http://{{ .Values.global.gateway.tls.host }}.{{ .Release.Namespace }}.svc.cluster.local:{{.Values.global.gateway.port }}/director/graphql"
+            - name: GRAPHQL_CLIENT_GRAPHQL_OAUTH_ENDPOINT # used when system-broker acts as an integration system, we need to convert the oauth2 opaque access token to jwt id token
```

Comment:
Why do we want to keep the integration system way of authenticating?

Reply:
Good point. I'll just comment out this env variable.

Reply:
My question is a more general one - why is there still code which allows for communicating with the System Broker as an integration system? There was this Token provider from secret that I saw was still in the code base. I think we decided that that would be removed all together

Reply:
We've discussed this topic with a small part of the team and we've decided to leave the integration system way of communicating because there's a possibility that we'll need this in the future. As of now, we're not 100% sure that we'll never this way of communication. So we designed the System Broker to work in both ways via token provider matching.

Reply:
We've cleared the dead code around the integration system flow of communicating.